### PR TITLE
Add greeting message and correct typo in README

### PR DESCRIPTION
**Pull request by strumerlz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and [git](https://github.com/Kottans/git-course) courses.
 You can try sending pull-requests to this repository proposing a change.
 The change can be anything you like, for example: fixing a typo in README.md. Consider this phrase, for example:
 
-*This was an intentionaly mispelld phrase*
+*This was an intentionaly misspelled phrase*
```
Comment (miroslav-kolomiets) [inline README.md:13]: Hi! Good job! But it's an example, revert it please.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (strumerlz) [reply README.md]: Oh, shame on me. Reverted it.

File: README.md [hunk 2]
```diff
@@ -1797,6 +1797,19 @@ P.S that's Riziy and he help me study ;)
 
 ![Riziy kitten](assets/images/riziy-vcheryasova.jpg)
 
+---
+
+#Hello kottans!
```
Comment (miroslav-kolomiets) [inline README.md:1802]:
```suggestion
# Hello kottans!
```
If you want it to be heading level 1, you need add space here.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Good! Please resolve conflict, and we will merge.

File: README.md (hunk 2 above)
Comment (strumerlz) [reply README.md]: Thanks for advice! Edited it

Comment (strumerlz) [reply README.md]: Oh, just realized that I could commit your suggestion instead of editing file by myself)

**Review by miroslav-kolomiets — APPROVED** [review]

**Pull request merged**